geoportal/c2cgeoportal_geoportal/scaffolds/create/tilegeneration/config.yaml.tmpl.mako_tmpl

```diff
@@ -101,6 +101,12 @@ generation:
     # maximum allowed consecutive errors, after it exit [default to 10]
     maxconsecutive_errors: 10
 
+sqs:
+    # The region where the SQS queue is
+    region: eu-west-1
+    # The SQS queue name, it should already exists
+    queue: <queue_name>
```

Comment:
You don't want to get that from the env?

Reply:
+1 :-)